<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-1RLFDVNI" failures="1" tests="2" name="Default test(failed)" time="0.027" errors="0" timestamp="10 May 2021 20:05:20 GMT">
  <testcase classname="testcases.RetryFailedDemo" name="test2" time="0.006"/>
  <testcase classname="testcases.RetryFailedDemo" name="test3" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testcases.RetryFailedDemo.test3(RetryFailedDemo.java:22)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="testcases.RetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
